### PR TITLE
fix bold or italic hyperlinks can't be opened through Cmd+Click

### LINKED ISSUE #1698: Can not open bold or italic hyperlinks through Cmd+Click
### Description
I just switched to MarkText from Typora, and I found that I can't open a Bold or Italic hyperlink text with cmd+click, maybe someone can tell me why? 

### Steps to reproduce
Add bold or italic style to a hyperlink text, and try to use cmd+click to open it.

**Expected behavior:**
Open the link in a browser.

**Actual behavior:**
Nothing happened.

### Versions
- Mark Text: v0.15.1
- Operating system: Mac OS X 10.15.1

By the way, thanks for building such a wonderful tool you guys, great work!

### PR DESCRIPTION
**Pull request by MrHeer**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes
| New feature?     | no
| BC breaks?       | no
| Deprecations?    | no
| New tests added? | not needed
| Fixed tickets    | Fixes #1698
| License          | MIT

### Description

fix bold or italic hyperlinks can't be opened through Cmd+Click

### CODE REVIEW

**Comment by MrHeer** [comment]
I know it ugly, but I can't find a better solution.

**Comment by Jocs** [comment]
@MrHeer Your PR solve this syntax `[**strong**]()`, it maybe also has other cases like:

```md
[`inline code`]()

[~~line through~~]()

[**`inline and strong`**]()
```

So, get parentNode is not enough?

File: src/muya/lib/contentState/clickCtrl.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ const clickCtrl = ContentState => {
     // format-click
     const node = selection.getSelectionStart()
     const inlineNode = node ? node.closest('.ag-inline-rule') : null
+    const parentNode = inlineNode ? inlineNode.parentNode : null
```
Comment (Jocs) [inline src/muya/lib/contentState/clickCtrl.js]: only get `parentNode` maybe not enough to find the `A` tag?

Comment (MrHeer) [reply src/muya/lib/contentState/clickCtrl.js]: Yes, let me see if there is any other way.

**Comment by Jocs** [comment]
@MrHeer Thanks for your contribution!

**Pull request merged**